### ISSUE TITLE
add requirements.txt

### ISSUE THREAD

**Issue by wisehackermonkey** [cduck/morse#3]
is it possible to add a requirements.txt to the codebase.
its not apparent which versions of the depenencies are required to run the code.
thanks

**Comment by cduck** [comment]
Yes, but it has been 5 years since I've used this code.  Let me know if you find the package version ranges this works with.  Ideally, I would repackage this for PyPI with the requirements in setup.py just like [this](https://github.com/cduck/bloch_sphere/).
